front/app/modules/commercial/flag_inappropriate_content/admin/components/ActivityWarningsTab.tsx

```diff
@@ -9,12 +9,6 @@ import { injectIntl } from 'utils/cl-intl';
 
 import messages from './messages';
 
-declare module 'modules/commercial/moderation/admin/containers/index' {
-  export interface ITabNamesMap {
-    warnings: 'warnings';
-  }
-}
-
```

Comment:
I don't think we needed this ITabNamesMap actually - when I removed it the inappropriate content tab still worked as intended (maybe since we had another ITabNamesMap in /front/app/containers/Admin/Moderation/index.tsx?)